core/vtk/ttkSignedDistanceField/ttkSignedDistanceField.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
```

Comment:
I'd recommend to replicate the header comments (documentation) from the base code header (also add a "\sa" doxygen command to the base code class)

Reply:
Done in the next commit!